### PR TITLE
Feature/drag and drop widget

### PR DESCRIPTION
**Pull request by Flight**
This PR implements drag and drop for the widgets in the structure mode.

Please **do not review dropzone.scss**.

**JS is also outdated**, just tried to leave the old styling (because it can be used by some other plugins).

### CODE REVIEW

File: filer/private/sass/layout/_main.scss [hunk 1]
```diff
@@ -566,3 +566,46 @@ body {
...
+    overflow: hidden;
+    background: #FFF url('/static/filer/icons/nofile_48x48.png');
+    background-size: cover;
+    border: solid 1px #ddd;
+    border-radius: 3px;
+    margin-right: 10px;
+    display: inline-block;
+    vertical-align: top;
+    img[src=''],
+    img:not([src]) {
```
Comment (Brankub) [inline filer/private/sass/layout/_main.scss]: Did you check if you can use this with oldest version of IE which you support?

Comment (Flight) [inline filer/private/sass/layout/_main.scss]: yes, we also using math calc in the admin styles.

**Comment by ruslanmalogulko** [comment]
:+1:

File: filer/private/sass/layout/_main.scss (hunk 1 above)
Comment (Brankub) [inline filer/private/sass/layout/_main.scss]: :+1:

**Pull request merged**